Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs

```diff
@@ -451,13 +451,14 @@ private void UserInteraction(IEntity player, GridCoordinates coordinates, Entity
 
             var item = hands.GetActiveHand?.Owner;
 
+            if(ActionBlockerSystem.CanChangeDirection(player))
+                playerTransform.LocalRotation = new Angle(coordinates.ToMapPos(_mapManager) - playerTransform.MapPosition.Position);
```

Comment:
Doesn't this change logic by being up here instead of down there?

Reply:
Not really. Before, it would only execute when CanInteract was true. Now, it only executes whenever CanChangeDirection is true, without taking CanInteract into account. This is useful for when players are knocked down, so they can change sprite orientation but not be able to interact with objects.